.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "trailingComma": "es5",
+  "tabWidth": 2,
+  "semi": false,
```

Comment:
I'd avoid this tbh, there are cases where semi quotes are required, see https://prettier.io/docs/en/rationale.html#semicolons